src/ontology/doid-edit.owl

```diff
@@ -40307,6 +40318,88 @@ AnnotationAssertion(rdfs:label obo:DOID_0060911 "karyomegalic interstitial nephr
+
+# Class: obo:DOID_0060916 (Proteasome-associated autoinflammatory syndrome 3)
+
+AnnotationAssertion(obo:IAO_0000115 obo:DOID_0060916 "Proteasome-associated autoinflammatory syndrome-3 is an autosomal recessive syndrome with onset in early infancy. Affected individuals present with nodular dermatitis, recurrent fever, myositis, panniculitis-induced lipodystrophy, lymphadenopathy, and dysregulation of the immune response, particularly associated with abnormal type I interferon-induced gene expression patterns. Additional features are highly variable, but may include joint contractures, hepatosplenomegaly, anemia, thrombocytopenia, recurrent infections, autoantibodies, and hypergammaglobulinemia. Some patients may have intracranial calcifications.
+
+https://pubmed.ncbi.nlm.nih.gov/?cmd=link&linkname=pubmed_pubmed&from_uid=26524591")
+AnnotationAssertion(oboInOwl:hasDbXref obo:DOID_0060916 "OMIM:617591")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:DOID_0060916 "PRAAS3")
+AnnotationAssertion(oboInOwl:hasOBONamespace obo:DOID_0060916 "disease_ontology")
+AnnotationAssertion(oboInOwl:id obo:DOID_0060916 "DOID:0060916")
+AnnotationAssertion(rdfs:label obo:DOID_0060916 "Proteasome-associated autoinflammatory syndrome 3"@en)
+SubClassOf(obo:DOID_0060916 obo:DOID_0050737)
+SubClassOf(obo:DOID_0060916 obo:DOID_0060919)
+SubClassOf(obo:DOID_0060916 ObjectSomeValuesFrom(obo:IDO_0000664 obo:GENO_0000930))
+
+# Class: obo:DOID_0060917 (facioscapulohumeral muscular dystrophy 3)
+
+AnnotationAssertion(dc:description obo:DOID_0060917 "Facioscapulohumeral muscular dystrophy-3 (FSHD3) is a digenic muscle disorder characterized by adult onset of proximal muscle weakness affecting the face, neck, scapular muscles, and upper and lower limbs. Muscle involvement is usually asymmetric, and other muscle groups may become involved with progression of the disease")
+AnnotationAssertion(oboInOwl:hasDbXref obo:DOID_0060917 "OMIM:619477")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:DOID_0060917 "FSHD3")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:DOID_0060917 "facioscapulohumeral muscular dystrophy type 3")
+AnnotationAssertion(oboInOwl:hasOBONamespace obo:DOID_0060917 "disease_ontology")
+AnnotationAssertion(oboInOwl:id obo:DOID_0060917 "DOID:0060917")
+AnnotationAssertion(rdfs:label obo:DOID_0060917 "facioscapulohumeral muscular dystrophy 3"@en)
+SubClassOf(obo:DOID_0060917 obo:DOID_11727)
+SubClassOf(obo:DOID_0060917 ObjectSomeValuesFrom(obo:IDO_0000664 obo:GENO_0000930))
+
+# Class: obo:DOID_0060918 (facioscapulohumeral muscular dystrophy 4)
+
+AnnotationAssertion(obo:IAO_0000115 obo:DOID_0060918 "Facioscapulohumeral muscular dystrophy (FSHD) is a skeletal muscle disorder characterized by adult onset of progressive muscle weakness of the face and upper extremity muscles. With disease progression, other muscles also may become affected. There is significant clinical variability and incomplete penetrance.
```

Comment:
Fix definition by reducing length and fitting to pattern: "A(n) {parent term} characterized by {symptoms, phenotypes, onset} that has_material_basis_in {genetic cause}."

Reply:
Changes have been implemented.